CodePen Links/LINKS.md

```diff
@@ -1,46 +1,3 @@
 # CodePen Links
-
 Welcome to our collection of amazing Frontend Wonders created on CodePen! Below, you'll find a list and table of incredible projects submitted by our contributors. Feel free to explore, get inspired, and don't forget to show some love by liking and commenting on these pens!
-
-## How to Add Your Link
-
-To add your CodePen link:
-
-1. Fork this repository
-2. Edit this file (`CodePen Links/LINKS.md`)
-3. Add your link to both the list and the table in the following formats:
-
-   For the list:
-
-   ```
-   - [Your Name or GitHub Username](Your CodePen Link) - Brief description of your Frontend Wonder
-   ```
-
-   For the table:
-
-   ```
-   | Your Name or GitHub Username | [Project Title](Your CodePen Link) | Brief description of your Frontend Wonder |
-   ```
-
-4. Submit a Pull Request
-
-## The Gallery of Frontend Wonders
-
-### List View
-
-<!-- Add your CodePen links below this line -->
-
-- [MohakBajaj](https://github.com/MohakBajaj) - [The Matrix Rain](https://mohakbajaj.github.io/The-Matrix-Rain/): A mesmerizing visual effect inspired by The Matrix movie, with customizable color and speed [^1]
-- [MohakBajaj](https://github.com/MohakBajaj) - [WebCam to ASCII](https://mohakbajaj.github.io/WebCam-to-ASCII/): A creative project that converts webcam frames to ASCII art using p5.js [^2]
-
-### Table View
-
-| Contributor                                 | Project                                                          | Description                                        |
-| ------------------------------------------- | ---------------------------------------------------------------- | -------------------------------------------------- |
-| [MohakBajaj](https://github.com/MohakBajaj) | [The Matrix Rain](https://mohakbajaj.github.io/The-Matrix-Rain/) | A mesmerizing visual effect inspired by The Matrix |
-| [MohakBajaj](https://github.com/MohakBajaj) | [WebCam to ASCII](https://mohakbajaj.github.io/WebCam-to-ASCII/) | Converts webcam frames to ASCII art using p5.js    |
-
-<!-- Add your CodePen links to the table below this line -->
-
-[^1]: https://github.com/MohakBajaj/The-Matrix-Rain
-[^2]: https://github.com/MohakBajaj/WebCam-to-ASCII/
+[jalaj-25](https://codepen.io/Jalaj-Singhal/pen/ZEgOpjQ) - In this i have made a front page for the restaurant 
```

Comment:
Please follow the guidelines and add to the table or list view only. I ask that you please not remove the GUIDELINES CONTENT.